toolkits/slack/evals/eval_chat.py

```diff
@@ -0,0 +1,1020 @@
+                },
+            ),
+        ],
+        critics=[
+            SimilarityCritic(critic_field="user_name", weight=0.75),
+            SimilarityCritic(critic_field="message", weight=0.25, similarity_threshold=0.5),
+        ],
+    )
+
+    suite.add_case(
+        name="Channel name similar to username",
+        user_message="Post 'sounds great!' in john-project channel",
+        expected_tool_calls=[
+            ExpectedToolCall(
+                func=send_message_to_channel,
+                args={
+                    "channel_name": "john-project",
+                    "message": "Sounds great!",
+                },
+            )
+        ],
+        critics=[
+            BinaryCritic(critic_field="channel_name", weight=0.5),
+            SimilarityCritic(critic_field="message", weight=0.5),
+        ],
+    )
+
+    return suite
+
+
+@tool_eval()
+def list_conversations_eval_suite() -> EvalSuite:
+    """Create an evaluation suite for tools listing conversations."""
+    suite = EvalSuite(
+        name="Slack Messaging Tools Evaluation",
+        system_message="You are an AI assistant that can interact with Slack to send messages and get information from conversations, users, etc.",
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    cases = [
+        (
+            "List my conversations",
+            "List all conversations I am a member of",
+            list_conversations_metadata,
+        ),
+        (
+            "List public channels",
+            "List all public channels",
+            list_public_channels_metadata,
+        ),
+        (
+            "List private channels",
+            "List all private channels",
+            list_private_channels_metadata,
+        ),
+        (
+            "List group direct message channels",
+            "List all group direct message channels",
+            list_group_direct_message_channels_metadata,
+        ),
+        (
+            "List individual direct message channels",
+            "List all individual direct message channels",
+            list_direct_message_channels_metadata,
+        ),
+        (
+            "List direct message channels",
+            "List all direct message channels",
+            list_direct_message_channels_metadata,
+        ),
+        (
+            "List public and private channels",
+            "List public and private channels I am a member of",
+            list_public_channels_metadata,
+            list_private_channels_metadata,
+        ),
+        (
+            "List public channels and direct message conversations",
+            "List public channels and direct message conversations I am a member of",
+            list_public_channels_metadata,
+            list_direct_message_channels_metadata,
+        ),
+    ]
+
+    for name, user_message, *expect_called_tool_functions in cases:
+        suite.add_case(
```

Comment:
there should be a critic to make sure the model isn't predicting something like "limit": 5 since these cases are trying to get all conversations.